lib/dealership.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/car'
+
+class Dealership
+  attr_reader :name, :address, :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+
+  end
+
+  def add_car(car)
+    @inventory.push(car)
+  end
+
+  def inventory
+    return @inventory
+  end
```

Comment:
You don't need these because you are using `attr_reader :inventory` which creates this method for us